index.html

```diff
@@ -4854,6 +4854,39 @@ <h2>
 (see the registration process for more detail) [[?DID-SPEC-REGISTRIES]].
       </p>
     </section>
+
+    <section>
+      <h2>
+Level of Assurance
+      </h2>
```

Comment:
```suggestion
      <h2>Level of Assurance</h2>
```
as had to be done to the other headings